core/templates/dev/head/domain/classifier/AnswerClassificationResultObjectFactory.js

```diff
@@ -19,17 +19,26 @@
 
 oppia.factory('AnswerClassificationResultObjectFactory', [function() {
   var AnswerClassificationResult = function(
-      outcome, answerGroupIndex, ruleIndex, classificationCategorization) {
+      labelledAsCorrect, outcome, answerGroupIndex, ruleIndex,
```

Comment:
labelledAsCorrect --> answerIsCorrect, ditto below. Use labelledAsCorrect only for the property of an answer group.